src/containers/widget/index.js

```diff
@@ -210,6 +210,7 @@ Widget.defaultProps = {
   isModalOpen: false,
   invoiceDetails: null,
   fetchingInvoiceState: false,
+  styles: "",
```

Comment:
question: styles will be a string by default?

Reply:
I thought I do it similar to the other props we have. 
from the server we either get nothing (the `styles` key will not be in the response) or a string with styles

Reply:
ok.